packages/jaeger-ui-components/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { default as TraceTimelineViewer } from './TraceTimelineViewer';
 export { default as TracePageHeader } from './TracePageHeader';
+export { default as SpanBarSettings } from './settings/SpanBarSettings';
```

Comment:
Wdyt about organizing this alongside the other generic trace configuration components like trace to logs? They're in`public/app/core/components`? Could create a Tracing subfolder and put TraceToLogs, TraceToMetrics, and SpanBarSettings there.

Reply:
Follow up PR? (want to keep this one small and specific)

Reply:
Yeah, sounds good to me. You good doing that or I can?

Reply:
I'll do it :)